sdk/core/amqp-common/src/requestResponseLink.ts

```diff
@@ -98,7 +98,7 @@ export class RequestResponseLink implements ReqResLink {
     if (!options) options = {};
 
     if (!options.timeoutInSeconds) {
-      options.timeoutInSeconds = 10;
+      options.timeoutInSeconds = Constants.defaultOperationTimeoutInSeconds;
```

Comment:
side note: This not only affects requests on the $cbs link, but also $management links